content/en/blog/_posts/2020-04-21-contributor-communication-upstream-marketing.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: blog
+title: How Kubernetes contributors are building a better communication process
+date: 2020-04-21
+slug: contributor-communication
+---
+
+**Authors:** Paris Pittman
+
+> “Perhaps we just need to use a different word. We may need to use community development or project advocacy as a word in the open source realm as opposed to marketing, and perhaps then people will realize that they need to do it.” 
+> ~ [*Nithya Ruff*](https://todogroup.org/www.linkedin.com/in/nithyaruff/) (from [*TODO Group*](https://todogroup.org/guides/marketing-open-source-projects/))
+
+A common way to participate in the Kubernetes contributor community is
+to be everywhere. 
+
+We have an active Slack, many mailing lists, Twitter account(s), and
+dozens of community-driven podcasts and newsletters that highlight all
+end-user, contributor, and ecosystem topics. And guess what else - an
+entire repository of amazing documentation that I bet there is one gem
```

Comment:
[an entire repository](https://github.com/kubernetes/community)

Reply:
it would be cool to link the repo here as well

Reply:
does "guess what else come off" as rude? that wasn't my intention at all - more cheery but I don't think it reads that way. worst case scenario, the sentence can lead without it.

Reply:
nit: I agree that this could be separated into two sentences.
Guess what ... -- a SIG and repository dedicated to ... Kubernetes documentation.

Reply:
Perhaps something like "And to add on top all of that, we also have repositories of amazing documentation, tons of meetings that drive the project forward, and recorded code deep dives." ?

Reply:
went with @jonasrosland suggestion and included some links ex: [meetings] with calendar - PR coming soon